### ISSUE TITLE
BENCHMARK MULTIPLE GPU TAKES TOO LONG!!!!

### ISSUE THREAD

**Issue by StartxPH** [nicehash/NiceHashMiner#883]
How do I set multiple GPU TO benchmark simultaneously? 
Its taking forever to benchmark a 10pcs gpu ....

please help..

2.0.1.8 version

**Comment by peteraclausen** [comment]
You need to switch to the latest NHML in order to benchmark GPU's in parallel

**Comment by StartxPH** [comment]
i am using the latest one that is on the website... and it does not bench all gpu at once...

**Comment by peteraclausen** [comment]
You have to run NHML 1.8.2.0-Pre2.  The latest version of the 1.8.1.x branch does not support it.

**Comment by Braintelligence** [comment]
@DillonN I would love the multi-benchmarking function to be merged into the newest version 😸 .

**Comment by S74nk0** [comment]
Issue not related to nhm/nhml client. Issue is related to discontinued nhm2 client.
